release-content/0.14/release-notes/12916_Implement_visibility_ranges_also_known_as_hierarchical_lev.md

```diff
@@ -1,4 +1,19 @@
 <!-- Implement visibility ranges, also known as hierarchical levels of detail (HLODs). -->
 <!-- https://github.com/bevyengine/bevy/pull/12916 -->
 
-<!-- TODO -->
+When looking at objects far away, it's hard to make out the details!
+This obvious fact is just as true in rendering as it is in real life.
+As a result, using complex, high-fidelity models for distant objects is a waste: we can replace their meshes with simplified equivalents (whose lower resolution textures are called mipmaps).
```

Comment:
```suggestion
As a result, using complex, high-fidelity models for distant objects is a waste: we can replace their meshes with simplified equivalents.
```

I don't think texture mipmaps are particularly relavant to LODs. While there are similarities in what they try to do, I don't think visibility ranges cares about mipmaps.